testhelper/convenience.go

```diff
@@ -33,10 +33,12 @@ func yellow(str interface{}) string {
 }
 
 func logFatal(t *testing.T, str string) {
+	t.Helper()
```

Comment:
Nice, but this should be a separate PR that we backport.

Reply:
Done #2997